init_x: 0  # initial x-pos 
init_y: 0  # initial y-pos
init_z: 50  # initial z-pos
init_psi: 0  # initial heading

max_omega: 1.0  # drone max angular velocity
max_zvel: 5  # drone max vertical velocity

vehicle_l: 3  # vehicle list for visualization
vehicle_num: $(arg vehicle_num)

hvel: 10  # vehicle x-y velocity
vvel: 2  # vehicle z-velocity

# number of random targets & obstalces
# -1 if providing custom list of targets
n_rand_targets: 10
n_rand_obst: 75

del_t: .02  # timestep between sim updates

waypt_threshold: 3.0

min_formation_dist: 50
max_formation_dist: 100

K_p: 10  # PID angular proportionality constant
K_p_z: 0.01  # PID z-axis proportionality constant

targets:
    - {id: 0, x: 51.551044627636315,  y: 149.8029287662543,  heading: 3.378180793959543, linear_speed: 0.0, angular_speed: 0.0, linear_speed_std: 0.0, angular_speed_std: 0.0}
    - {id: 1, x: 10.974144160580153,  y: 30.040757437172914,  heading: 4.508843153595021, linear_speed: 0.0, angular_speed: 0.0, linear_speed_std: 0.0, angular_speed_std: 0.0}
    - {id: 2, x: 0.5680554085412552,  y: -2.664048919196034,  heading: 4.672608384214387, linear_speed: 0.0, angular_speed: 0.0, linear_speed_std: 0.0, angular_speed_std: 0.0}
    - {id: 3, x: 156.8464087510376,  y: -211.03092801199222,  heading: 0.30215158640044787, linear_speed: 0.0, angular_speed: 0.0, linear_speed_std: 0.0, angular_speed_std: 0.0}
    - {id: 4, x: -160.88091999108138,  y: 181.3921102556269,  heading: 1.1403573889711118, linear_speed: 0.0, angular_speed: 0.0, linear_speed_std: 0.0, angular_speed_std: 0.0}
    - {id: 5, x: -80.69773372799929,  y: -203.699142226512,  heading: 0.5070628325206488, linear_speed: 0.0, angular_speed: 0.0, linear_speed_std: 0.0, angular_speed_std: 0.0}
    - {id: 6, x: 177.2074932012082,  y: -8.196788203943271,  heading: 5.038209188479517, linear_speed: 0.0, angular_speed: 0.0, linear_speed_std: 0.0, angular_speed_std: 0.0}
    - {id: 7, x: 157.66459842688243,  y: 159.37324549272654,  heading: 4.081182485142326, linear_speed: 0.0, angular_speed: 0.0, linear_speed_std: 0.0, angular_speed_std: 0.0}
    - {id: 8, x: 24.669747468224273,  y: 26.568956842395806,  heading: 4.437959305981657, linear_speed: 0.0, angular_speed: 0.0, linear_speed_std: 0.0, angular_speed_std: 0.0}
    - {id: 9, x: -162.68783111131904,  y: -90.7619464737769,  heading: 2.9456967368377964, linear_speed: 0.0, angular_speed: 0.0, linear_speed_std: 0.0, angular_speed_std: 0.0}


obstacles:
    - {id: 75, x: 190.15103035660877,  y: 139.61160151039246,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 74, x: -205.2458906892181,  y: 206.18523669873844,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 73, x: -82.64452246337027,  y: 70.50047989030338,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 72, x: 33.59670356924275,  y: -51.79166202803921,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 71, x: -204.76713454258368,  y: 142.0517352734651,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 70, x: -14.351803072317011,  y: -40.17234114100114,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 69, x: -33.953799412004514,  y: 56.936016292525494,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 68, x: 33.03004720207187,  y: 8.6663610668692,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 67, x: -208.59057807748607,  y: -174.59584095127894,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 66, x: 75.7791190627828,  y: -160.02125593665173,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 65, x: 194.69759227476436,  y: 212.5101378888616,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 64, x: 123.56848806682711,  y: 139.7931836736148,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 63, x: -168.79532178653633,  y: 144.3666208288925,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 62, x: -209.76767523653896,  y: 214.23435500788224,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 61, x: -13.179980239318155,  y: 137.62716712538145,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 60, x: -200.46644058019612,  y: -178.67985470741434,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 59, x: -31.59555872342014,  y: -79.61523758169761,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 58, x: 208.9194313920957,  y: -191.5957365221006,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 57, x: 182.45668825734919,  y: 80.87953250315485,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 56, x: 161.48634169904585,  y: -48.59997812034942,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 55, x: -69.76535285525435,  y: 136.16416922463662,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 54, x: 102.71218900686426,  y: -11.49068242491012,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 53, x: -19.648289226769407,  y: -131.14521992619925,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 52, x: -126.20774609909952,  y: -58.41454855338213,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 51, x: -140.91262499659732,  y: -19.181638533725817,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 50, x: 46.32491611045663,  y: -76.29723674256593,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 49, x: 74.75888454939565,  y: -106.2408608351357,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 48, x: -203.98333343516225,  y: 140.18098569010044,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 47, x: 200.66554988084266,  y: 66.62619156506327,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 46, x: 137.33549355176615,  y: -139.68708481783378,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 45, x: -139.9177787363219,  y: -22.207402032295306,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 44, x: 196.71137932639652,  y: -91.7601606227716,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 43, x: -122.8589016969478,  y: -177.4534158486357,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 42, x: -97.49819906741504,  y: 135.37701904552398,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 41, x: -143.02738707088812,  y: 94.99511684983929,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 40, x: 20.02589388682236,  y: 181.73429129587117,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 39, x: 106.76964223908493,  y: 104.76129723842132,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 38, x: 106.85783155606765,  y: 95.24897903990893,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 37, x: -69.84469523474297,  y: 64.2239191726689,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 36, x: 42.8090808836464,  y: -205.60067908958678,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 35, x: 12.385271578852667,  y: 98.70401310353776,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 34, x: -201.80407906698642,  y: 31.38219293684122,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 33, x: 134.984929594638,  y: 195.42391684017036,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 32, x: -59.424225100485245,  y: 2.6902732553529063,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 31, x: -72.22594716833535,  y: 162.4626216354945,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 30, x: 0.6291658617855092,  y: -31.783333289671532,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 29, x: 109.42589267504599,  y: -144.29568656233596,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 28, x: -85.71681743211195,  y: 110.44444164554278,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 27, x: -207.2301420877385,  y: -38.19142835507702,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 26, x: -98.09773193813226,  y: 86.48791550624799,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 25, x: -213.83760477128928,  y: -75.1018849064917,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 24, x: 155.7546841263154,  y: 76.27579643277329,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 23, x: 135.11699146726914,  y: 92.08273570725464,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 22, x: 90.66911300895003,  y: 129.94763523714113,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 21, x: 106.62532418744428,  y: -17.280102531003962,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 20, x: 182.7045760002556,  y: -195.8874259395305,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 19, x: 44.76537610552964,  y: -56.372877056719204,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 18, x: -41.37517910553427,  y: 171.18737444414387,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 17, x: 151.3625563675232,  y: 172.19668744113324,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 16, x: -180.50153661103917,  y: -121.51931191307754,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 15, x: -146.96886190752878,  y: 14.424683007886046,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 14, x: -185.56139845002934,  y: 62.0158009058141,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 13, x: -16.860296230250384,  y: 42.08210550791205,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 12, x: 201.2515170608251,  y: -60.39981385222504,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 11, x: 26.20022740816418,  y: 80.27162019502748,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 10, x: -96.11896482587113,  y: 190.02730929786856,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 9, x: -166.3881910755693,  y: 95.9890154231465,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 8, x: 195.4114253400187,  y: -217.80767885211117,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 7, x: -27.900557941052796,  y: 12.404898288809022,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 6, x: -103.55914460669683,  y: 99.863425639083,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 5, x: -196.11572109275264,  y: 27.573480285230517,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 4, x: -206.11249906032708,  y: 197.7005003843031,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 3, x: 58.90068091958244,  y: 186.18035225721917,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 2, x: -108.19202330308508,  y: -202.51295032258042,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 1, x: -113.32537118579747,  y: 80.36942261163176,  width: 5.0, length: 5.0, height: 100.0, speed: 0.0, hold_heading_time: 1.0}


sensor_a: 1
sensor_b: 1
sensor_d: 1
sensor_g: 1
sensor_h: 1

sensor_focal_length: 8
sensor_width: 40
sensor_height: 20
sensor_pitch: 1.57  # 0.349
sensor_yaw: 1.57
